### PR TITLE
Release to maven central

### PR DESCRIPTION
**Pull request by Thoymas**
Maven and travis-ci related configuration changes to enable releases to maven central through OSSRH.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -278,3 +280,38 @@ Make sure to set the timeoutSeconds to that of your slowest test in the test sui
...
+The above steps are enough for deploying to sonatype/maven central.
+Depending on the version number in the pom, the build artifact will be deployed to either the snapshots repository or the staging-repository.
+
+
+Playbook for making a release
+-----------------------------
+Basically follow this guide: http://central.sonatype.org/pages/apache-maven.html#performing-a-release-deployment
+
+* Change the version number to the release version you want. Should not include -SNAPSHOT in the name.
+* Commit, tag with release number, push
```
Comment (PelleUllberg) [inline README.md]: We could as well add git cli example here

File: README.md [hunk 2]
```diff
@@ -278,3 +280,38 @@ Make sure to set the timeoutSeconds to that of your slowest test in the test sui
...
+* Travis builds and deploys
+* Follow the http://central.sonatype.org/pages/releasing-the-deployment.html guide to promote the staged release to maven central.
+* Change version number to next snapshot version
+* Commit and push
```
Comment (PelleUllberg) [inline README.md]: And here

**Comment by PelleUllberg** [comment]
Nice work @Thoymas! Looks solid!

File: README.md (hunk 1 above)
Comment (Thoymas) [inline README.md]: Done.

File: README.md (hunk 2 above)
Comment (Thoymas) [inline README.md]: Done.

**Pull request merged**